BranchSDK/src/BranchIO/Util/WindowsStorage.cpp

```diff
@@ -132,6 +132,31 @@ WindowsStorage::set(const std::string& key, const std::string& value, Scope scop
     return *this;
 }
 
+bool
+WindowsStorage::getBoolean(const std::string& key, bool& value, Scope scope) const {
+    string registryKey, registryPath;
+    bool validKey(getRegistryKeyAndPath(scope, key, registryKey, registryPath));
+    assert(validKey);
```

Comment:
This isn't a test class, so assert looks out of place. Also, you already check for the validity of the key below (i.e. `if (!has(key, scope)) return false;`)

Reply:
The original implementation of storage has a ton of `assert`s sprinkled in.

I created a follow-up ticket `SDK-638` to resolve these.

Reply:
👍